Gemfile

```diff
@@ -3,7 +3,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.2'
 
-gem 'wdm', '>= 0.1.0'
+# gem 'wdm', '>= 0.1.0'
```

Comment:
I prefer to remove unused gems

Reply:
Another teammate need it in order to work with rails. But I had to comment it because it was giving me an error